packages/extension-sdk/src/connect/tile/types.ts

```diff
@@ -0,0 +1,230 @@
+   */
+  querySlug?: string
+  /**
+   * The filters being applied to the dashboard. If the tile
+   * is being configured as an explore this will not be populated.
+   */
+  dashboardFilters?: Filters
+  /**
+   * Indicates whether the dashboard is running. If the tile
+   * is being configured as an explore the state will be UNKNOWN.
+   * Note that for dashboard performance reasons, the runstate
+   * may NEVER be shown as running. This generally will happen
+   * if there no other tiles associated with a query (including
+   * the one the extension is associated with).
+   * If the extension needs to know for certain that a dashboard
+   * has been run, detecting differences in the lastRunStartTime
+   * is the reliable way.
+   */
+  dashboardRunState?: DashboardRunState
+  /**
+   * When true, the dashboard is being edited. If the tile
+   * is being configured as an explore this will not be populated.
+   */
+  isDashboardEditing?: boolean
+  /**
+   * When true, cross filtering. If the tile
+   * is being configured as an explore this will not be populated.
+   */
+  isDashboardCrossFilteringEnabled?: boolean
+  /**
+   * A query object that matches the query ID associated with the
+   * underlying dashboard element with any dashboard filters and
+   * timezone changes made at the dashboard level applied.
+   */
+  filteredQuery?: IQuery
+  /**
+   * The id of the tile extension element that triggered the last
+   * dashboard run. The id will be undefined if the dashboard run
+   * was triggered by the dashboard run button or auto run or if
+   * the run was triggered using the embed SDK. If the tile
+   * is being configured as an explore this will not be populated.
+   * Note that the lastRunSourceElementId CAN be the same as the
+   * element id of the current extension instance, in other words,
+   * if the extension triggers a dashboard run, it will be notified
+   * when the dashboard run starts and finishes.
+   */
+  lastRunSourceElementId?: string
+  /**
+   * Indicates the last dashboard run start time. If the tile
+   * is being configured as an explore this will not be populated.
+   * Note that the start and end times reported should not
+   * used for capturing performance metrics.
+   */
+  lastRunStartTime?: number
+  /**
+   * Indicates the last dashboard run end time. If the tile
+   * is being configured as an explore this will not be populated.
+   * If the tile is running, this will not be populated.
+   * Note that the start and end times reported should not
+   * used for capturing performance metrics.
+   */
+  lastRunEndTime?: number
+  /**
+   * Indicates whether the last dashboard run was succesful or not.
+   * If the tile is being configured as an explore this will not be
+   * populated.
+   * If the tile is running, this will not be populated.
+   */
+  lastRunSuccess?: boolean
+}
+
+export interface Pivot {
+  key: string
+  is_total: boolean
```

Comment:
you're using camelCase for other interface properties. Is this a copy from the Looker API?

Reply:
No. This was based upon the custom viz api